GI01SUMO/GI01SUMO_R00002_ingest.csv

```diff
@@ -1,86 +1,86 @@
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl16/ctdbp2/*ctdbp.log,GI01SUMO-RID16-03-CTDBPF000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/instruments/dcl16/CTDBP_sn16-50061/SBE*.hex,GI01SUMO-RID16-03-CTDBPF000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl16/velpt/*.velpt*.log,GI01SUMO-RID16-04-VELPTA000,recovered_host,Available,
+#mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/instruments/dcl16/VELPT_snAQD12198/*.aqd,GI01SUMO-RID16-04-VELPTA000,recovered_inst,Not Expected,Instrument didn't log internally
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl16/pco2w/*.pco2w.log,GI01SUMO-RID16-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/instruments/dcl16/PCO2W_snC0100/*.txt,GI01SUMO-RID16-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl16/dosta2/*dosta2.log,GI01SUMO-RID16-06-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.nutnr_b.dcl_full.nutnr_b_dcl_full_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl16/nutnr2/*.nutnr2.log,GI01SUMO-RID16-07-NUTNRB000,recovered_host,Available,
+mi.dataset.driver.nutnr_b.nutnr_b_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/instruments/dcl16/NUTNR_sn271/*.DAT,GI01SUMO-RID16-07-NUTNRB000,recovered_inst,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl16/spkir2/*.spkir2.log,GI01SUMO-RID16-08-SPKIRB000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl11/imm/adcp/adcp_*.DAT,GI01SUMO-RII11-02-ADCPSN010,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/instruments/dcl11/imm/ADCP_sn22865/_RDI*.000,GI01SUMO-RII11-02-ADCPSN010,recovered_inst,Available,
+#mi.dataset.driver.ctdbp_p.ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/instruments/dcl11/imm/CTDBP_sn16-50059/SBE*.hex,GI01SUMO-RII11-02-CTDBPP031,recovered_inst,Not Expected,Instrument lost
+mi.dataset.driver.ctdbp_p.dcl.ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/cg_data/dcl11/imm/ctdbp01/*.DAT,GI01SUMO-RII11-02-CTDBPP031,recovered_host,Available,
+mi.dataset.driver.ctdbp_p.ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00002/instruments/dcl11/imm/CTDBP_sn16-50073/SBE*.hex,GI01SUMO-RII11-02-CTDBPP032,recovered_inst,Available,

```

Comment:
something weird with ctdbp02: the data files in cg_data/dcl11/imm/ctdbp02 have SN 50069 up til ctdbp02_20150731_103028.dat then a few weird files and then at 20150815_142704.dat the SN changes to 50073?  Instruments/dcl11/imm/CTDBP_sn16-50073 files have sn 50073? Deploy csv and config sheet have 50073.

Reply:
ahhh maybe the ctdbp was swapped out prior to deployment, because 20150815 is the deploy date for the mooring.